go/vt/vttablet/tabletserver/connpool/pool.go

```diff
@@ -70,15 +71,17 @@ type Pool struct {
 // to publish stats only.
 func NewPool(env tabletenv.Env, name string, cfg tabletenv.ConnPoolConfig) *Pool {
 	idleTimeout := cfg.IdleTimeoutSeconds.Get()
+	refreshTimeout := cfg.RefreshTimeoutSeconds.Get()
 	cp := &Pool{
 		env:                env,
 		name:               name,
 		capacity:           cfg.Size,
 		prefillParallelism: cfg.PrefillParallelism,
 		timeout:            cfg.TimeoutSeconds.Get(),
 		idleTimeout:        idleTimeout,
+		refreshTimeout:     refreshTimeout,
 		waiterCap:          int64(cfg.MaxWaiters),
-		dbaPool:            dbconnpool.NewConnectionPool("", 1, idleTimeout, 0),
+		dbaPool:            dbconnpool.NewConnectionPool(name+"_dbaPool", 1, idleTimeout, 0, 0),
```

Comment:
```suggestion
		dbaPool:            dbconnpool.NewConnectionPool(name+"_dbaPool", 1, idleTimeout, refreshTimeout, 0),
```

Is there special reason for not refreshing `dbaPool`?

Reply:
@jjh-kim is it same with below comments?